mafia/models.py

```diff
@@ -159,10 +158,12 @@ def last_victim_name(self):
         """Make last_victim_name queryable."""
         return self.property3
 
-
     def fail_bystander_vectors(self):
-        mafiosi = Node.query.filter_by(network_id=self.id,
-                                       property2='True', type='mafioso').all()
+        """As implemented, this fails Vectors of both Bystanders (non-Mafia),
+        and dead Mafia.
+        TODO: Is this what we want?
```

Comment:
In general, the logic around failing things is pretty murky, and it seems relevant to the problems we're pursuing. Some effort to clarify the code in these cases would be worthwhile.